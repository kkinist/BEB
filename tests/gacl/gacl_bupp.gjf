%chk=gacl.chk
%nprocs=4
%mem=1000mw
# HF/GEN gfinput geom=check IOP(6/81=3) pseudo=read scf=xqc pop(all,thresh=1)

BEB step 3: pseudopotential B and U for gacl

0 1

Cl Ga 0
SDDall
****
Cl 0
 D   1 1.00       0.000000000000
      7.5000000000e-01  0.1000000000D+01
****
Ga 0
 D   1 1.00       0.000000000000
      1.6900000000e-01  0.1000000000D+01
****

Cl Ga 0
SDDall

