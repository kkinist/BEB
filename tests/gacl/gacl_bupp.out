 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-29317.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=     29319.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=gacl.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ----------------------------------------------------------------------
 # HF/GEN gfinput geom=check IOP(6/81=3) pseudo=read scf=xqc pop(all,th
 resh=1)
 ----------------------------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,17=8,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 6/7=2,8=2,9=2,10=2,28=1,81=3,87=-1,88=1/1;
 99/5=1,9=1/99;
 --------------------------------------------
 BEB step 3: pseudopotential B and U for gacl
 --------------------------------------------
 Structure from the checkpoint file:  "gacl.chk"
 Charge =  0 Multiplicity = 1
 Redundant internal coordinates found in file.
 Ga,0,0.,0.,-0.0254811392
 Cl,0,0.,0.,2.2254811392
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1         31           0        0.000000    0.000000   -0.025481
      2         17           0        0.000000    0.000000    2.225481
 ---------------------------------------------------------------------
 Stoichiometry    ClGa
 Framework group  C*V[C*(ClGa)]
 Deg. of freedom     1
 Full point group                 C*V     NOp   4
 Largest Abelian subgroup         C2V     NOp   4
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1         31           0        0.000000    0.000000    0.797216
      2         17           0        0.000000    0.000000   -1.453746
 ---------------------------------------------------------------------
 Rotational constants (GHZ):      0.0000000      4.2994334      4.2994334
 General basis read from cards:  (5D, 7F)
 ======================================================================================================
                                       Pseudopotential Parameters
 ======================================================================================================
  Center     Atomic      Valence      Angular      Power
  Number     Number     Electrons     Momentum     of R      Exponent        Coefficient   SO-Coeffient
 ======================================================================================================
    1         31            3
                                      G and up 
                                                     2        1.0000000        0.00000000    0.00000000
                                      S - G
                                                     2        5.2159600      203.85397200    0.00000000
                                      P - G
                                                     2        4.3089040      156.10339000    0.00000000
                                      D - G
                                                     2        0.4963570        1.03164700    0.00000000
                                      F - G
                                                     2        1.7151700      -10.67373500    0.00000000
    2         17            7
                                      F and up 
                                                     2        1.0000000        0.00000000    0.00000000
                                      S - F
                                                     2        6.3943000       33.13663200    0.00000000
                                                     2        3.1971000       16.27072800    0.00000000
                                      P - F
                                                     2        5.6207000       24.41699300    0.00000000
                                                     2        2.8103000        7.68305000    0.00000000
                                      D - F
                                                     2        5.3381000       -8.58764900    0.00000000
 ======================================================================================================
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   3 1.00       0.000000000000
      0.3484775000D+01  0.1884253118D+00
      0.1685835000D+01 -0.5931936637D+00
      0.1785420000D+00  0.1142798511D+01
 S   1 1.00       0.000000000000
      0.6781400000D-01  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.1283231000D+01  0.1878902467D+00
      0.4761630000D+00 -0.2574158868D+00
      0.1556210000D+00 -0.8721126584D+00
 P   1 1.00       0.000000000000
      0.5429400000D-01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.1690000000D+00  0.1000000000D+01
 ****
      2 0
 S   3 1.00       0.000000000000
      0.1407307600D+02  0.4190225216D-01
      0.2331565000D+01 -0.5956792861D+00
      0.5071000000D+00  0.1298294273D+01
 S   1 1.00       0.000000000000
      0.1824330000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.3353129000D+01 -0.4592773552D-01
      0.7856860000D+00  0.4418444459D+00
      0.2674540000D+00  0.6541530079D+00
 P   1 1.00       0.000000000000
      0.7827500000D-01  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.1547700000D-01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.7500000000D+00  0.1000000000D+01
 ****

 There are    15 symmetry adapted cartesian basis functions of A1  symmetry.
 There are     2 symmetry adapted cartesian basis functions of A2  symmetry.
 There are     7 symmetry adapted cartesian basis functions of B1  symmetry.
 There are     7 symmetry adapted cartesian basis functions of B2  symmetry.
 There are    13 symmetry adapted basis functions of A1  symmetry.
 There are     2 symmetry adapted basis functions of A2  symmetry.
 There are     7 symmetry adapted basis functions of B1  symmetry.
 There are     7 symmetry adapted basis functions of B2  symmetry.
    29 basis functions,    47 primitive gaussians,    31 cartesian basis functions
     5 alpha electrons        5 beta electrons
       nuclear repulsion energy         4.9368758805 Hartrees.
 NAtoms=    2 NActive=    2 NUniq=    2 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
   1 Symmetry operations used in ECPInt.
 ECPInt:  NShTT=      66 NPrTT=     202 LenC2=      67 LenP2D=     202.
 LDataN:  DoStor=T MaxTD1= 6 Len=  172
 NBasis=    29 RedAO= T EigKep=  9.14D-02  NBF=    13     2     7     7
 NBsUse=    29 1.00D-06 EigRej= -1.00D+00 NBFU=    13     2     7     7
 ExpMin= 1.55D-02 ExpMax= 1.41D+01 ExpMxC= 1.41D+01 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (SG) (SG) (PI) (PI) (SG)
       Virtual   (PI) (PI) (SG) (PI) (PI) (SG) (SG) (PI) (PI) (SG)
                 (PI) (PI) (DLTA) (DLTA) (PI) (PI) (SG) (DLTA)
                 (DLTA) (PI) (PI) (SG) (SG) (SG)
 The electronic state of the initial guess is 1-SG.
 Keep R1 ints in memory in symmetry-blocked form, NReq=930923.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RHF) =  -16.8418908329     A.U. after   10 cycles
            NFock= 10  Conv=0.15D-08     -V/T= 3.6385
 QCSCF skips out because SCF is already converged.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (SG) (SG) (PI) (PI) (SG)
       Virtual   (PI) (PI) (SG) (PI) (PI) (SG) (SG) (PI) (PI) (PI)
                 (PI) (SG) (DLTA) (DLTA) (PI) (PI) (SG) (DLTA)
                 (DLTA) (PI) (PI) (SG) (SG) (SG)
 The electronic state is 1-SG.
 Alpha  occ. eigenvalues --   -1.04712  -0.54112  -0.43934  -0.43934  -0.36023
 Alpha virt. eigenvalues --    0.02171   0.02171   0.04701   0.05730   0.05730
 Alpha virt. eigenvalues --    0.11671   0.31026   0.34628   0.34628   0.37350
 Alpha virt. eigenvalues --    0.37350   0.38313   0.41953   0.41953   0.45814
 Alpha virt. eigenvalues --    0.45814   0.62193   1.18558   1.18558   1.24427
 Alpha virt. eigenvalues --    1.24427   1.27681   4.65648   8.32674
          Condensed to atoms (all electrons):
               1          2
     1  Ga   2.385081   0.166989
     2  Cl   0.166989   7.280940
 Mulliken charges:
               1
     1  Ga   0.447929
     2  Cl  -0.447929
 Sum of Mulliken charges =   0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  Ga   0.447929
     2  Cl  -0.447929
 Electronic spatial extent (au):  <R**2>=            112.1846
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              3.2543  Tot=              3.2543
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.2821   YY=            -21.2821   ZZ=            -28.1129
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              2.2769   YY=              2.2769   ZZ=             -4.5539
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=             31.8352  XYY=              0.0000
  XXY=              0.0000  XXZ=             11.7132  XZZ=              0.0000  YZZ=              0.0000
  YYZ=             11.7132  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -34.4710 YYYY=            -34.4710 ZZZZ=           -256.9580 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=            -11.4903 XXZZ=            -47.7299 YYZZ=            -47.7299
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 4.936875880494D+00 E-N=-4.428633764696D+01  KE= 6.383104812021D+00
 Symmetry A1   KE= 3.162339230029D+00
 Symmetry A2   KE= 2.325170669267D-51
 Symmetry B1   KE= 1.610382790996D+00
 Symmetry B2   KE= 1.610382790996D+00

 Atomic contributions to Alpha molecular orbitals:
 Alpha occ 1 OE=-1.047 is Cl2-s=0.96 Ga1-s=0.02 Ga1-p=0.01
 Alpha occ 2 OE=-0.541 is Cl2-p=0.55 Ga1-s=0.39 Cl2-s=0.03 Ga1-p=0.03
 Alpha occ 3 OE=-0.439 is Cl2-p=0.96 Ga1-p=0.03 Ga1-d=0.02
 Alpha occ 4 OE=-0.439 is Cl2-p=0.96 Ga1-p=0.03 Ga1-d=0.02
 Alpha occ 5 OE=-0.360 is Ga1-s=0.56 Cl2-p=0.26 Ga1-p=0.17 Ga1-d=0.01
 Alpha vir 6 OE=0.022 is Ga1-p=0.68 Cl2-p=0.32
 Alpha vir 7 OE=0.022 is Ga1-p=0.68 Cl2-p=0.32
 Alpha vir 8 OE=0.047 is Cl2-p=1.01 Ga1-s=-0.01
 Alpha vir 9 OE=0.057 is Cl2-p=0.69 Ga1-p=0.31
 Alpha vir 10 OE=0.057 is Cl2-p=0.69 Ga1-p=0.31
 Alpha vir 11 OE=0.117 is Ga1-p=0.92 Ga1-s=0.05 Cl2-s=-0.04 Ga1-d=0.04 Cl2-p=0.03
 Alpha vir 12 OE=0.310 is Cl2-p=0.95 Ga1-d=0.10 Ga1-p=-0.04 Cl2-s=-0.03 Ga1-s=0.01
 Alpha vir 13 OE=0.346 is Ga1-p=0.51 Cl2-p=0.46 Ga1-d=0.02
 Alpha vir 14 OE=0.346 is Ga1-p=0.51 Cl2-p=0.46 Ga1-d=0.02
 Alpha vir 15 OE=0.373 is Cl2-p=0.57 Ga1-p=0.42
 Alpha vir 16 OE=0.373 is Cl2-p=0.57 Ga1-p=0.42
 Alpha vir 17 OE=0.383 is Ga1-p=0.86 Ga1-d=0.07 Cl2-p=0.03 Cl2-d=0.03 Ga1-s=0.01
 Alpha vir 18 OE=0.420 is Ga1-d=1.00
 Alpha vir 19 OE=0.420 is Ga1-d=1.00
 Alpha vir 20 OE=0.458 is Ga1-d=0.92 Ga1-p=0.05 Cl2-d=0.04
 Alpha vir 21 OE=0.458 is Ga1-d=0.92 Ga1-p=0.05 Cl2-d=0.04
 Alpha vir 22 OE=0.622 is Ga1-d=0.77 Cl2-p=0.13 Ga1-p=0.06 Cl2-d=0.04 Cl2-s=-0.01 Ga1-s=0.01
 Alpha vir 23 OE=1.186 is Cl2-d=1.00
 Alpha vir 24 OE=1.186 is Cl2-d=1.00
 Alpha vir 25 OE=1.244 is Cl2-d=0.95 Ga1-d=0.04
 Alpha vir 26 OE=1.244 is Cl2-d=0.95 Ga1-d=0.04
 Alpha vir 27 OE=1.277 is Cl2-d=0.92 Ga1-p=0.05 Ga1-d=0.02 Ga1-s=0.01
 Alpha vir 28 OE=4.656 is Cl2-s=1.07 Ga1-p=-0.05 Cl2-p=0.03 Ga1-s=-0.03 Ga1-d=-0.02
 Alpha vir 29 OE=8.327 is Ga1-s=0.98 Cl2-s=0.03
 
 Orbital energies and kinetic energies (alpha):
                                 1                 2
   1         O                -1.047121          0.491534
   2         O                -0.541115          0.580921
   3         O                -0.439344          0.805191
   4         O                -0.439344          0.805191
   5         O                -0.360226          0.508715
   6         V                 0.021713          0.187687
   7         V                 0.021713          0.187687
   8         V                 0.047007          0.064388
   9         V                 0.057296          0.132137
  10         V                 0.057296          0.132137
  11         V                 0.116706          0.318774
  12         V                 0.310263          0.499391
  13         V                 0.346277          0.643127
  14         V                 0.346277          0.643127
  15         V                 0.373497          0.605591
  16         V                 0.373497          0.605591
  17         V                 0.383130          0.724529
  18         V                 0.419535          0.594347
  19         V                 0.419535          0.594347
  20         V                 0.458144          0.700764
  21         V                 0.458144          0.700764
  22         V                 0.621925          1.089758
  23         V                 1.185583          2.625191
  24         V                 1.185583          2.625191
  25         V                 1.244270          2.647125
  26         V                 1.244270          2.647125
  27         V                 1.276814          2.680946
  28         V                 4.656475          3.540900
  29         V                 8.326739          1.481316
 Total kinetic energy from orbitals= 6.383104812021D+00

 Test job not archived.
 1\1\GINC-GAMBA\SP\RHF\Gen\Cl1Ga1\IRIKURA\17-May-2017\0\\# HF/GEN gfinp
 ut geom=check IOP(6/81=3) pseudo=read scf=xqc pop(all,thresh=1)\\BEB s
 tep 3: pseudopotential B and U for gacl\\0,1\Ga,0,0.,0.,-0.0254811392\
 Cl,0,0.,0.,2.2254811392\\Version=ES64L-G09RevD.01\State=1-SG\HF=-16.84
 18908\RMSD=1.462e-09\Dipole=0.,0.,-1.2803296\Quadrupole=1.6928529,1.69
 28529,-3.3857059,0.,0.,0.\PG=C*V [C*(Cl1Ga1)]\\@


 Discoveries are often made by not following instructions,
 by going off the main road, by trying the untried.
 -- Frank Tyger
 Job cpu time:       0 days  0 hours  0 minutes 12.8 seconds.
 File lengths (MBytes):  RWF=      5 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 13:28:08 2017.
